app/js/components/orderEntry/SortAndFilter.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import sortAndFilterAction from '../../actions/sortAndFilterAction';
+
+const SortAndFilter = props => (
+  <div className="sort-filter-section">
+    <div className="sort-type">
+      <label className="label-sort">Type:</label>
+      <select
+        className="sort-select"
+        onChange={event => props.sortAndFilterAction('type', event.target.value)}>
+        Type:
+        <option value="all">All</option>
+        <option value="drugorder">Drug Orders</option>
+        <option value="testorder">Test Orders</option>
+      </select>
+    </div>
+    <div className="sort-status">
+      <label className="label-sort">Status:</label>
+      <select className="sort-select">
+        <option>All</option>
+        <option>Active</option>
+        <option>Inactive</option>
+      </select>
+    </div>
```

Comment:
Please add some comments(inline or block) as to why 
```
<div className="sort-status">
  ....
</div>
```
is hidden in the view.

Reply:
addressed